strategies.py

```diff
@@ -26,10 +26,10 @@
 
 import go
 
-MAX_GAME_DEPTH = int(go.N * go.N * 1.25)
 # When to do deterministic move selection.  ~30 moves on a 19x19, ~8 on 9x9
 TEMPERATURE_CUTOFF = int((go.N * go.N) / 12)
 
+MAX_DEPTH = (go.N ** 2) * 1.4  # ~500 moves for 19x19, ~110 for 9x9
```

Comment:
500 Seems kinda small. I've definitely seen early-phase minigo games last longer than 500 moves. Maybe it should be a bit larger, like 1000?

Reply:
AGZ used 2 * N^2. The point isn't to let all games gracefully finish, though - it's explicitly cutting games off to save computation, cull the weird tail-end of such games from the training data, and as a failsafe for weird infinite loop situations. (I wouldn't be surprised if a longer game end led to worse training because one player accidentally suicides his whole-board group.)

Reply:
Why don't we just use 2*N^2 then, if AGZ used that ?

Reply:
Because it really affects how slow things are at the beginning.  A lot  I'm happy to go ahead and say there aren't really meaningful go games longer than 400 moves, and 500 moves vs 720 moves is basically like a whole game's worth of wasted work.